android-pdf-viewer/src/main/java/com/github/barteksc/pdfviewer/util/PublicFunction.kt

```diff
@@ -0,0 +1,17 @@
+package com.github.barteksc.pdfviewer.util
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.BitmapFactory
+import java.io.ByteArrayOutputStream
+
+class PublicFunction {
```

Comment:
I removed the file but the function is in `AnnotationManager`. Do I remove the function too?

Reply:
> Do I remove the function too?

I think I mentioned this in another discussion but since we are not doing any immage annotation adding we can remove that

Reply:
It's removed